lib/Raven/Client.php

```diff
@@ -540,15 +540,16 @@ protected function getUserData()
     {
         $user = $this->context->getUserData();
         if (empty($user)) {
-            if (!function_exists('session_id') || !session_id()) {
```

Comment:
Please, notice this change

Reply:
What's the intended purpose of this?

Reply:
I think field `user` could exist even without logging. Anyway, in HTTP(S) requests we should save users's IP address

Reply:
This whole method will be useless after #487 will be merged because the user data will be retrieved from a middleware and the [`session_id`](http://php.net/manual/en/function.session-id.php) function won't be used anymore anyway

Reply:
Okay, please be sure you will test IPv4 and IPv6 as well (i.e. `8.8.8.8`, `2001:db8:0:0:0:0:2:1`, `2001:db8::2:1`)